src/xcode/ENA/ENA/Source/Scenes/FriendsInvite/FriendsInviteController.swift

```diff
@@ -79,11 +79,15 @@ final class FriendsInviteController: UIViewController {
 
 extension FriendsInviteController: UIActivityItemSource {
 	func activityViewControllerPlaceholderItem(_: UIActivityViewController) -> Any {
-		shareTitle
+		return shareTitle
```

Comment:
Do you prefer the `return`?

Reply:
I don't, but the whole project has it, so I change these too, to be consistent.